.github/workflows/pull-request.yaml

```diff
@@ -146,80 +146,45 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: set cache keys
```

Comment:
Our cache is causing false positives and negatives during our tests, causing our tests not to capture changes that should break tests.

This allowed us to merge a PR with a false negative test (the PR that changed the behavior of the Trace tab) and blocked this PR with a false positive.